Gemfile.lock

```diff
@@ -62,14 +62,14 @@ GEM
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.0.4)
       loofah (~> 2.2, >= 2.2.2)
-    railties (5.2.1)
-      actionpack (= 5.2.1)
-      activesupport (= 5.2.1)
+    railties (5.2.2.1)
+      actionpack (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       method_source
       rake (>= 0.8.7)
       thor (>= 0.19.0, < 2.0)
-    rake (12.3.1)
-    thor (0.20.0)
+    rake (12.3.2)
+    thor (0.20.3)
```

Comment:
### [thor](http://whatisthor.com/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.20.0 → 0.20.3

[_~~change-log~~, [source-code](https://github.com/wycats/thor)_]





<details>
<summary>Commits</summary>

A change of **73** commits. See the full changes on [the compare page](https://github.com/erikhuda/thor/compare/v0.20.0...v0.20.3).

These are the first 10 commits:
- (7bb6f63) [Release should use invoke not execute](https://github.com/erikhuda/thor/commit/7bb6f63ac8460fe5d738412ae6fede69084c4d07)
- (c44218b) [Add more tests for HashWithIndifferentAccess#fetch](https://github.com/erikhuda/thor/commit/c44218b73271b35bcafd7516b370018dcca5a575)
- (bbfdf33) [Merge pull request #568 from segiddins/seg-hash-fetch-tests](https://github.com/erikhuda/thor/commit/bbfdf33c2fb45ae72933cb4ebeb7c744a86ba6b2)
- (1d81e25) [Introduce a constant for the default terminal width](https://github.com/erikhuda/thor/commit/1d81e25b4b3d0c22d2b38710b0b43836deae5148)
- (c3bb746) [Merge pull request #572 from sschuberth/master](https://github.com/erikhuda/thor/commit/c3bb74614ea74fd6a249357ff6c51e48c1bd1233)
- (ac177a8) [require open-uri when loading http template](https://github.com/erikhuda/thor/commit/ac177a8a10ae007817c0cdee46515aee45e8eb4b)
- (e41ff11) [Merge pull request #576 from sshaw/master](https://github.com/erikhuda/thor/commit/e41ff11dbfcfa0a01422adb47f991a74e72e964e)
- (1e1d85d) [Fix incorrect use of Process::exit. This fixes open issue #2…](https://github.com/erikhuda/thor/commit/1e1d85dbfd5ecb56e8a4bfc29767614284924794)
- (2644b85) [lock hashdiff to <0.3.6 to fix travis](https://github.com/erikhuda/thor/commit/2644b853e5870ba8d27e8cbc5fbdfb6ae9a236c2)
- (cb3bb7d) [update ruby version in travis config](https://github.com/erikhuda/thor/commit/cb3bb7dbd968c0ab8b65b1059e77e58e0fb890fa)


</details>